proprietary/Components/src/denhaag/accordion.tokens.json

```diff
@@ -0,0 +1,22 @@
+{
+  "denhaag": {
+    "accordion": {
+      "border": { "value": "1px solid {denhaag.color.grey.2.value}" },
+      "font-family": { "value": "{denhaag.typography.sans-serif.font-family.value}" },
+      "font-weight": { "value": "400" },
+      "line-height": { "value": "24px" },
+      "padding": { "value": "16px" },
+      "margin": { "value": "12px" },
+      "expanded": {
+        "font-weight": { "value": "500" },
```

Comment:
Today i learned when the font shows like this in Figma,
![Screenshot 2021-12-09 at 12 31 02](https://user-images.githubusercontent.com/94894596/145420891-a7ac80f0-9dfa-45c1-a7d4-b8f270d5595b.png)

The font-weight is 700.

Reply:
Updated in commit: https://github.com/nl-design-system/denhaag/pull/600/commits/1ae898c3f293a8c5c8e5862c65c9f3ec4c1e8d25